packages/cfd/src/Containers/compare-accounts-modal.tsx

```diff
@@ -101,18 +55,9 @@ const CompareAccountsModal = ({
 }: TCompareAccountsModalProps) => {
     const show_eu_related = (is_logged_in && is_eu) || (!is_logged_in && is_eu_country);
     const is_dxtrade = platform && platform === CFD_PLATFORMS.DXTRADE;
-    const mt5_accounts = [
-        landing_companies?.mt_gaming_company?.financial,
-        landing_companies?.mt_financial_company?.financial,
-        landing_companies?.mt_financial_company?.financial_stp,
-    ];
-
-    const cfd_account_button_label =
-        mt5_accounts.filter(Boolean).length === 1 || (is_demo_tab && platform === CFD_PLATFORMS.DXTRADE)
-            ? localize('Account Information')
-            : localize('Compare accounts');
```

Comment:
Why are we removing this? for logged out dashboard, we still need to display `Compare accounts` as the title.

Reply:
@matin-binary This component is only used in the Deriv X dashboard and this dashboard is not available for logged-out users. This condition is used for MT5 Dashboard.

Reply:
<img width="814" alt="Screenshot 2022-10-04 at 2 53 26 PM" src="https://user-images.githubusercontent.com/70938039/193754298-25f61e11-f2e4-486c-be79-3f45cda1a7b5.png">
Button title got affected by your changes.

Reply:
Yes, you're right. I fixed this.